static/styles/dark_theme.css

```diff
@@ -464,6 +464,11 @@
         border-color: hsla(0, 0%, 0%, 0.9);
     }
 
+    textarea.new_message_textarea.invalid,
+    textarea.new_message_textarea.invalid:focus {
+        border-color: hsl(0, 58.3%, 27.3%);
```

Comment:
I'd love color suggestions -- I picked something for light and dark mode but am not confident in them and I'm especially unfamiliar with the way dark mode errors usually look.